storage/include/libstqe.sh

```diff
@@ -51,8 +51,11 @@ function stqe_init_fwroot
     for cmd in python python3 python2; do
         $cmd -V > /dev/null 2>&1 && python=$cmd && break
     done
-    [[ -n $python ]] || cki_skip_task "python not found"
-
+    if [[ -z $python ]]; then
+        cki_run_cmd_pos "yum install -y python36 python3-pip python3-setuptools" || \
+            cki_abort_task "fail python not found"
```

Comment:
Why `fail python not found` instead of simply `python not found`? Is it that you want to search for the string `fail`? If so, and if case sensitiveness doesn't matter, maybe it would look nicer/clearer if it read `FAIL: python not found`. If not, maybe drop the `fail` altogether?

Reply:
I'm trying to keep fail messages formatting the same through the whole script but this reformatting of messages can be the next commit. Yes, we are using these fail messages to better track the failures.